code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -92,6 +92,17 @@
 	flags_inv = HIDEFACE
 	body_parts_covered = 0
 
+/obj/item/clothing/mask/trinary_mask
+	name = "trinary perfection mask"
+	desc = "A simple lace mask worn by IPCs and organics alike while within the churches of the Trinary Perfection."
+	icon_state = "trinary_mask"
+	item_state = "trinary_mask"
+	w_class = 2
```

Comment:
```suggestion
	w_class = ITEMSIZE_SMALL
```

You reverted this change for some raisin?

Reply:
Your thing is outdated sir!

Reply:
I blame github's website... somehow. :+1: